app/javascript/components/subject/Jurisdiction.js

```diff
@@ -0,0 +1,202 @@
+              <Form.Group className="px-4">
+                <Form.Check
+                  type="radio"
+                  className="mb-1"
+                  name="apply_to_group"
+                  id="apply_to_group_no"
+                  label="This monitoree only"
+                  onChange={this.handleChange}
+                  checked={!this.state.apply_to_group}
+                />
+                <Form.Check
+                  type="radio"
+                  className="mb-3"
+                  name="apply_to_group"
+                  id="apply_to_group_yes"
+                  label="This monitoree and all household members"
+                  onChange={this.handleChange}
+                  checked={this.state.apply_to_group}
+                />
+              </Form.Group>
+            </React.Fragment>
+          )}
+          <Form.Group>
+            <Form.Label>Please include any additional details:</Form.Label>
+            <Form.Control as="textarea" rows="2" id="reasoning" onChange={this.handleChange} />
+          </Form.Group>
+        </Modal.Body>
+        <Modal.Footer>
+          <Button variant="secondary btn-square" onClick={toggle}>
+            Cancel
+          </Button>
+          <Button variant="primary btn-square" onClick={submit} disabled={this.state.loading}>
+            {this.state.loading && (
+              <React.Fragment>
+                <span className="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>&nbsp;
+              </React.Fragment>
+            )}
+            Submit
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <div className="disabled">
+          <Form.Label className="nav-input-label">
+            ASSIGNED JURISDICTION
+            <InfoTooltip tooltipTextKey="assignedJurisdiction" location="right"></InfoTooltip>
+          </Form.Label>
+          <Form.Group className="d-flex mb-0">
+            <Form.Control
+              as="input"
+              id="jurisdiction_id"
+              list="jurisdictionPaths"
+              autoComplete="off"
+              className="form-control-lg"
+              onChange={this.handleChange}
+              onKeyPress={this.handleKeyPress}
+              value={this.state.jurisdiction_path}
+            />
+            <datalist id="jurisdictionPaths">
+              {Object.entries(this.props.jurisdictionPaths).map(([id, path]) => {
+                return (
+                  <option value={path} key={id}>
+                    {path}
+                  </option>
+                );
+              })}
+            </datalist>
+            {!this.state.validJurisdiction || this.state.jurisdiction_path === this.props.jurisdictionPaths[this.state.original_jurisdiction_id] ? (
+              <Button className="btn-lg btn-square text-nowrap ml-2" disabled>
```

Comment:
Same feedback as elsewhere about just putting this check on the `disabled` prop.

Reply:
done